webpack.config.dev.js

```diff
@@ -34,7 +34,7 @@ export default {
 		new webpack.DefinePlugin({
 			"process.env": {
 				"NODE_ENV": JSON.stringify("development"),
-				"CLOWDER_REMOTE_HOSTNAME": JSON.stringify(process.env.CLOWDER_REMOTE_HOSTNAME),
+				"CLOWDER_REMOTE_HOSTNAME": JSON.stringify(process.env.CLOWDER_REMOTE_HOSTNAME || "http://localhost:8000"),
```

Comment:
I copied this from INCORE because when no "remote hostname" get passed in, then it will just be "" and hence use relative URL. It will make automatic deployment easier. 

I'm suggesting moving this to script section of `package.json`:
`"start:dev": "export CLOWDER_REMOTE_HOSTNAME=http://localhost:8000 && npm run start",`
then instead of `npm run`, do `npm run:dev`

Reply:
Works great, thanks!